packages/web-components/@sfx/sayt/test/unit/sayt.test.ts

```diff
@@ -22,6 +23,17 @@ describe('Sayt Component', () => {
       expect(sayt).to.be.an.instanceof(LitElement);
     });
 
+    it('should debounce certain methods', () => {
```

Comment:
This test is currently testing the internals of `setDebounce()`. The test should only assert that it was called (what the `constructor` does), not what happens inside of it.